pocketmine.json

```diff
@@ -0,0 +1,36 @@
+{
+    "PocketMine": {
+        "containers": {
+            "pocketmine": {
+                "image": "magicalyak/pocketmine",
+                "launch_order": 1,
+                "opts": [
+		  [
+			"-it",
+			"--restart=always",
```

Comment:
-it should be removed it's not needed and I need to set resets to 1 or 2 it always needs one restart. Will fix soon